pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -229,179 +231,139 @@ func (bd *backupDriver) monitorBackups(ctx context.Context) {
 			continue
 		}
 		var backupCount int
-		backupCountQuery := fmt.Sprintf(`SELECT count(DISTINCT end_time) FROM [SHOW BACKUP FROM LATEST IN %s]`, bd.sp.scheduledBackupSpecs.backupCollection())
+		backupCountQuery := fmt.Sprintf(`SELECT count(DISTINCT end_time) FROM [SHOW BACKUP FROM LATEST IN '%s']`, fixtureURI.String())
 		sql.QueryRow(bd.t, backupCountQuery).Scan(&backupCount)
 		bd.t.L().Printf(`%d scheduled backups taken`, backupCount)
-		if backupCount >= bd.sp.scheduledBackupSpecs.numBackupsInChain {
+		if backupCount >= bd.sp.fixture.IncrementalChainLength {
 			pauseSchedulesQuery := fmt.Sprintf(`PAUSE SCHEDULES WITH x AS (SHOW SCHEDULES) SELECT id FROM x WHERE label = '%s'`, scheduleLabel)
 			sql.QueryRow(bd.t, pauseSchedulesQuery)
 			break
 		}
 	}
 }
 
+func newFixtureRegistry(ctx context.Context, t test.Test, c cluster.Cluster) *blobfixture.Registry {
+	var uri url.URL
+	switch c.Cloud() {
+	case spec.AWS:
+		uri = url.URL{
+			Scheme:   "s3",
+			Host:     "cockroach-fixtures-us-east-2",
+			RawQuery: "AUTH=implicit",
+		}
+	case spec.GCE, spec.Local:
+		account, err := vm.Providers["gce"].FindActiveAccount(t.L())
+		require.NoError(t, err)
+		t.L().Printf("using GCE account", account)
+
+		uri = url.URL{
+			Scheme:   "gs",
+			Host:     "cockroach-fixtures-us-east1",
+			RawQuery: "AUTH=implicit",
+		}
+	default:
+		t.Fatalf("fixtures not supported on %s", c.Cloud())
+	}
+
+	uri.Path = path.Join(uri.Path, "roachtest/v25.1")
```

Comment:
Perhaps instead the subfolder we merge is "master". Then, we can put in a check that if the test runs on a release branch, it fails if the subfolder is master. This would trigger a failure every time a new branch is cut, which would remind us to trivially backport a dir change.